plottable.d.ts

```diff
@@ -1415,7 +1415,7 @@ declare module Plottable {
 
 
 declare module Plottable {
-    type AnchorCallback = (component: Component) => any;
```

Comment:
Should this actually be called `ComponentCallback`?  Do we want to use this signature for anything other than anchoring?